### ISSUE TITLE
Dataset and VariantSet name not set.

### ISSUE THREAD

**Issue by jeromekelleher** [ga4gh/server#648]
The Dataset name is not set. Running
```
$ python client_dev.py datasets-search http://localhost:8000
```
gives 
```
MWtnLXAzLXN1YnNldA==    None
``` 
The expected output is 
```
MWtnLXAzLXN1YnNldA==  1kg-p3-subset
```

Similarly for VariantSets --- the name should be the directory name.

**Comment by jeromekelleher** [comment]
Again, this example is using the updated client tools in #643.

**Comment by dcolligan** [comment]
Fixed in #651
